catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/DescribeCommandTest.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.catalog;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+import java.io.IOException;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.source.CatalogProvider;
+
+public class DescribeCommandTest {
+
+    private static final String TEST_CATALOG_PROVIDER_TITLE = "test catalog provider title";
+
+    private static final String TEST_CATALOG_PROVIDER_DESCRIPTION =
+            "test catalog provider description";
+
+    private static final String TEST_CATALOG_PROVIDER_ID = "test catalog provider id";
+
+    private static final String TEST_CATALOG_PROVIDER_VERSION = "test catalog provider version";
+
+    private static final String TEST_CATALOG_FRAMEWORK_TITLE = "test catalog framework title";
+
+    private static final String TEST_CATALOG_FRAMEWORK_DESCRIPTION =
+            "test catalog framework description";
+
+    private static final String TEST_CATALOG_FRAMEWORK_ID = "test catalog framework id";
+
+    private static final String TEST_CATALOG_FRAMEWORK_VERSION = "test catalog framework version";
+
+    private DescribeCommand describeCommand;
+
+    private ConsoleOutput consoleOutput;
+
+    @Before
+    public void setUp() throws Exception {
+        consoleOutput = new ConsoleOutput();
```

Comment:
I updated the unit tests to extend a common `AbstractConsoleOutputTest` class to include these before/after methods for the consoleOutput.